### DISCUSSION TITLE
Favor extender en la plataforma RLXMOOC las fechas para labs 5,6,7

### DISCUSSION THREAD

**Discussion by admaesmo** [rramosp/ai4eng.v1#331] [UDEA Students]
Hola, la plataforma de calificación automatica RLXMOOC no tiene las fechas modificadas para el 02 de noviembre como se acordó la semana pasada, por favor extender las fechas en dicha plataforma.
Muchas gracias.

**Comment by MateoBlandonMesa** [comment]
Profe, estoy teniendo este error al momento de enviar las actividades del lab 6:
"error: end_date (set session.debug=True for tracebacks)"
Creo que es porque no se ha actualizado la fecha de entrega.

<img width="463" alt="Screenshot_1" src="https://github.com/user-attachments/assets/7cb22179-2847-4fea-b95f-6cca41107620">
